README.md

```diff
@@ -0,0 +1,1075 @@
+# node-diacritics-transliterator
+
+> Diacritic transliteration tools using diacritics.io API
+
+# Installation
+
+```bash
+$ npm install diacritics-transliterator
+```
+
+# Usage
+
+```js
+const diacritics = require("diacritics-transliterator");
+const de = diacritics.getData({
+    language: "German"
+});
+/* de => {"de":{...}} */
+```
+
+# API
+
+## Contents
+
+- [Initializing](#initializing)
+- [Basic functions](#basic-functions)
+  - [getVersion](#getversion)
+  - [setVersion](#setversion)
+  - [getData](#getdata)
+  - [getDiacritics](#getdiacritics)
+  - [formatUnicode](#formatunicode)
+- [Transliteration functions](#transliteration-functions)
+  - [transliterate](#transliterate)
+  - [createRegExp](#createregexp)
+  - [replacePlaceholder](#replaceplaceholder)
+
+## Initializing
+
+Each API method requires the `diacritics-transliterator` module.
+
+Use `new` to instantiate the module.
+
+**Syntax**
+
+```js
+const diacritics = require("diacritics-transliterator");
+```
+
+**Return Value**
+
+The diacritics module.
+
+**Parameters**
+
+<dl>
+  <dt><code>version</code> (optional)</dt>
+  <dd>The version of the database API to use. Defaults to the current version.</dd>
+</dl>
+
+## Basic functions
+
+### getVersion
+
+Use this function to get the currently set API version.
+
+**Syntax**
+
+```js
+diacritics.getVersion();
+```
+
+**Return Value**
+
+The database API major release version formatted as a string with a "v" prefix.
+
+**Parameters**
+
+None
+
+**Example**
+
+```js
+const diacritics = require("diacritics-transliterator");
+const version = diacritics.version;
+/* version => "v1" */
+```
+
+### setVersion
+
+Upon initialization, this module is set to the latest major release version of the diacritics API. To change the version, set the version of the API to the desired major release.
+
+**Syntax**
+
+```js
+diacritics.version = version;
+```
+
+**Return Value**
+
+If valid, the set database API version formatted as a string with a "v" prefix.
+
+If invalid, the current database API major release version is returned.
+
+**Parameters**
+
+<dl>
+  <dt><code>version</code></dt>
+  <dd>The version to set. Formatted as a string <code>"v1"</code> or number <code>1</code>.</dd>
+</dl>
+
+**Example**
+
+```js
+const diacritics = require("diacritics-transliterator");
+diacritics.version = "v1";
+/* => "v1" */
+```
+
+### getData
+
+Function to return specificially filtered data from the diacritics database. Include one or more filters to narrow the query. Invalid filters are ignored.
+
```

Comment:
It does support filtering by an array [now](https://github.com/diacritics/api/commit/5bcc0c50cdb5189e4ed154d0ebe8b40409811bf9).